lib/Service/ShareService.php

```diff
@@ -85,7 +93,9 @@ public function findAll(string $nodeType, int $nodeId, ?string $userId = null, b
 		$userId = $this->permissionsService->preCheckUserId($userId);
 
 		try {
-			$shares = $this->mapper->findAllSharesForNode($nodeType, $nodeId, $userId);
+			$excluded = !$this->circleHelper->isCirclesEnabled() ? ['circle'] : [];
```

Comment:
Another nitpick: ideal turn `'cirlce'` into a constant (and adapt follow up usages).

(Yes, existing `group` string suffers from the same)